DOC-4278 (#86)
exoplatform/exo-documentation: docs/Manage-Space.rst

1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.

> 2. A popup is displayed containing all informations needed to create the space.

Reviewer: ```suggestion
2. A popup is displayed containing all information needed to create the space.
```

as suggested by @mzorai  ;-)

> 2. A popup is displayed containing all information related to the space creation.

Reviewer: ```suggestion
2. A popup is displayed containing all informations needed to create the space.
```

Reviewer 2: @plamarque the word information is uncountable and should never take an "s" in the end

.. _access-level-step:

> A pulldown lists all predefined templates that the space creator can use. A description of each template is displayed below the selectors.

Reviewer: ```suggestion
A pulldown lists all predefined templates that the space creator can use. A description of each template is displayed below.
```

.. note:: Each template defines its default visibility, registration criteria and list of applications.

.. _About-space-visibility:

4. The space creator may override the predefined visibility and registration options of the space.

> -  **Visibility**: By default, the Visibility value is set to be the one predefined in the template.

Reviewer: ```suggestion
-  **Hidden**: The visibility of the space is predefined by the template and can be overridden at creation time.
```

> 	-  No: The space is not hidden and always visible in the public spaces list.

Reviewer: ```suggestion
	-  No: The space is visible in the public spaces directory
```

> 	-  Yes: The space is not visible in the public spaces list.

Reviewer: ```suggestion
	-  Yes: The space is hidden, hence it won't appear in the public spaces directory.
```

> -  **Registration**: By default, the Registration value is set to be the one predefined in the template.

Reviewer: ```suggestion
-  **Registration**: The Registration policy is predefined by the template and can be overridden at creation time
```

   -  Close: The user cannot request for joining, but only the space
      manager can invite him.

> 5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.

Reviewer: ```suggestion
5. In the **Users ** field, the space creator can invite users or space members to the space just by picking them with the selector.
```

Type-ahead suggestions allow to facilitate selection users and groups to invite.

> .. note:: Only spaces that the creator is member of can be selected.

Reviewer 2: ```suggestion
.. note:: Only spaces where the creator is member could be selected.
```

> 6. Click Create Space button to finish adding your new space. The new space appears.

Reviewer: @hghorbel  we should have some style  for representing product labels such as button names. @mzorai what do you suggest ?

Reviewer 2: we can use these quotes ``Create Space``, they will appear in red

Reviewer 2: looks like the expression was interpreted. In fact we use the quotes `` around the word (with alt gr + 7 of the keyborad)

> **Sapce banner and avatar**

Reviewer: ```suggestion
**Space banner and avatar**
```

>    By default, the space banner is set to be the one predefined in the template and the space

Reviewer: ```suggestion
   By default, the space banner is predefined by the template and the space
```

>    avatar contains the general space image:

Reviewer: ```suggestion
   avatar contains the generic space image:
```
